examples/libvirt-k8s-upstream.env

```
@@ -0,0 +1,11 @@
+# This is an example configuration file for a upstream Kubernetes deployment
+# on libvirt
+#
+# For a full list of configuration options, see the files in
+# config/*.toml
+
+export ROOKCHECK_HARDWARE_PROVIDER="LIBVIRT"
+export ROOKCHECK_LIBVIRT_CONNECTION="qemu:///system"
+export ROOKCHECK_LIBVIRT_VM_MEMORY="8"
+export ROOKCHECK_NODE_IMAGE_USER=opensuse
+export ROOKCHECK_LIBVIRT_IMAGE = "http://download.opensuse.org/distribution/leap/15.1/jeos/openSUSE-Leap-15.1-JeOS.x86_64-OpenStack-Cloud.qcow2"
```

the extra spaces around ' = ' need to be removed otherwise this produces a syntax error.
```suggestion
export ROOKCHECK_LIBVIRT_IMAGE="http://download.opensuse.org/distribution/leap/15.1/jeos/openSUSE-Leap-15.1-JeOS.x86_64-OpenStack-Cloud.qcow2"
```